devel/qt6-qtcreator/Portfile

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           qt6 1.0
+
+name                qt6-qtcreator
+
+version             15.0.0
+revision            0
+categories          devel aqua
+# from qt6 PortGroup:
+# qt 6.8 - darwin >= 20
+# qt 6.4 - darwin >= 18
+platforms           {darwin >= 18}
+# from https://blog.qt.io/blog/2016/01/13/new-agreement-with-the-kde-free-qt-foundation/
+# The GPL license comes with two exceptions that ensure that there are no license restrictions on generated code, and that bridging to 3rd party code is still possible.
+license             GPL-3
+maintainers         {mcalhoun @MarcusCalhoun-Lopez} openmaintainer
```

Comment:
I know @MarcusCalhoun-Lopez was the qt/qt5-creator maintainer, but this is new, we probably shouldn't sign him up.